HyperV/scripts/gerrit-git-prep.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash -e

```

Comment:
Tested with latest upstream and did not managed to make it work; instead it worked with the current one.